lib/simplefootball_web_web/router.ex

```diff
@@ -1,26 +1,20 @@
 defmodule SimplefootballWebWeb.Router do
   use SimplefootballWebWeb, :router
 
-  pipeline :browser do
-    plug :accepts, ["html"]
-    plug :fetch_session
-    plug :fetch_flash
-    plug :protect_from_forgery
-    plug :put_secure_browser_headers
-  end
-
   pipeline :api do
     plug :accepts, ["json"]
   end
 
-  scope "/", SimplefootballWebWeb do
-    pipe_through :browser
+  # Other scopes may use custom stacks.
+  scope "/api", SimplefootballWebWeb do
+    pipe_through :api
+
+    resources "/competitions", CompetitionController, only: [:index]
 
-    get "/", PageController, :index
+    get(
+      "/competitions/:competitionType/currentMatchday",
```

Comment:
Nitpick: Ich mag hyphen-case in URLs. Das ist auch was Google empfiehlt. Also `/current-matchday`